castoredc_api/client/castoredc_api_client.py

```diff
@@ -1,9 +1,10 @@
 """Module for interacting with the Castor EDC API."""
+
+import asyncio
 import csv
 from datetime import datetime
 from itertools import chain
-import asyncio
-from typing import Optional, List, Union
+from typing import List, Optional, Union
```

Comment:
I have format-on-save enabled and isort installed. If this is really undesirable I can rebase this out.